docs/source/02_installation.md

```diff
@@ -1,4 +1,4 @@
-# Building ADAM from Source {#build-from-source}
+# Building ADAM from Source
```

Comment:
I didn't think this was the correct formatting for these anchors, based on [adam docs](http://adam.readthedocs.io/en/latest/source/02_installation/). There seemed to be two issues:

* The `{#build-from-source}` shows up on the page, which looks weird. 
* Links to such anchors don't work, for example http://adam.readthedocs.io/en/latest/source/02_installation/#python-build just goes to the top of the page.

Since you can link to headers with just the text of the header, separated by dashes (for example, http://adam.readthedocs.io/en/latest/source/02_installation/#running-adam), I just removed the anchors and updated existing links to this format.